plugins/modules/foreman_hostgroup.py

```diff
@@ -237,8 +287,13 @@ def main():
             config_groups=dict(type='entity_list', flat_name='config_group_ids'),
             puppet_proxy=dict(type='entity', flat_name='puppet_proxy_id'),
             puppet_ca_proxy=dict(type='entity', flat_name='puppet_ca_proxy_id'),
+            openscap_proxy=dict(type='entity', flat_name='openscap_proxy_id'),
             parameters=dict(type='nested_list', entity_spec=parameter_entity_spec),
+            content_source=dict(type='entity', flat_name='content_source_id'),
+            lifecycle_environment=dict(type='entity', flat_name='lifecycle_environment_id'),
+            content_view=dict(type='entity', flat_name='content_view_id'),
         ),
+        argument_spec=dict(organization=dict()),
```

Comment:
Just thinking: Does this render it a (fake) `KatelloEntity{Apypie}AnsibleModule`? Or is the `organization` required there?

Reply:
In a "real" Katello module, the Org would be required, yes.